package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "triplesec",
-  "version": "3.0.27",
+  "version": "3.1.0",
```

Comment:
not sure if anything else needs to be done other than this to bump. also, not sure if we should bump to 4.0 because the default is now version 4?

Reply:
Yes, that sounds good.  I just need to run `mwr -I` when I release it.